nixos/tests/turn-rs.nix

```
@@ -0,0 +1,65 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  {
+    name = "turn-rs";
+
+    nodes = {
+      server = {
+        virtualisation.vlans = [ 1 ];
+
+        networking = {
+          useNetworkd = true;
+          useDHCP = false;
+          firewall.enable = false;
+        };
+
+        systemd.network.networks."01-eth1" = {
+          name = "eth1";
+          networkConfig.Address = "10.0.0.1/24";
+        };
+
+        services.turn-rs = {
+          enable = true;
+          secretFile = pkgs.writeText "secret" ''
+            USER_1_CREDS="foobar"
+          '';
+          settings = {
+            turn = {
+              realm = "localhost";
+              interfaces = [
+                {
+                  transport = "udp";
+                  bind = "127.0.0.1:3478";
+                  external = "127.0.0.1:3478";
+                }
+                {
+                  transport = "tcp";
+                  bind = "127.0.0.1:3478";
+                  external = "127.0.0.1:3478";
+                }
+              ];
+            };
+
+            auth.user1 = "$USER_1_CREDS";
```

Ah, on a final note - the 3.0.1 -> 3.1.0 bump brought a change to this. It should now look like this:

```suggestion
            auth.static_credentials.user1 = "$USER_1_CREDS";
```